Reaktoro/Core/Phases.test.cxx

```diff
@@ -429,4 +429,122 @@ TEST_CASE("Testing Phases", "[Phases]")
         checkAqueousPhase(phasevec[0], "H2O(aq) H+ OH- H2(aq) O2(aq) CO2(aq) HCO3- CO3--");
         checkGaseousPhase(phasevec[1], "CO2(g) O2(g) H2(g) H2O(g) CH4(g) CO(g)");
     }
+
+    //=================================================================================================================
+    //-----------------------------------------------------------------------------------------------------------------
+    // TESTING CLASS: AqueousPhase with provided speciates and tags, so that species possessing them are excluded
+    //-----------------------------------------------------------------------------------------------------------------
+    //=================================================================================================================
+    SECTION("Testing AqueousPhase::AqueousPhase(Speciate, Exclude)")
+    {
+        Database db("supcrt98.yaml");
```

Comment:
Here, too, these calls to Database(supcrt98.yaml) are slowing down the execution of the tests. Please use the `db` object above.

Reply:
Corrected